src/main/cli/cli.c

```diff
@@ -2758,6 +2738,7 @@ static void cliName(char *cmdline)
         if (strncmp(cmdline, emptyName, len)) {
             strncpy(pilotConfigMutable()->name, cmdline, MIN(len, MAX_NAME_LENGTH));
         }
+        cliPrintLine("###WARNING: This command will be removed. Use 'set name = ' instead.###");
```

Comment:
We put other error messages _after_ the offending line - we probably should do this here too, for consistency.

Reply:
Updated warning to be at the end.
```
# name MyQuad

# name: MyQuad
###WARNING: This command will be removed. Use 'set name = ' instead.###
```